prac_05/state_names.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
```

Comment:
Great dictionary name! Should the dictionary also include South Australia?

Reply:
Good catch 👍